src/MediaManagerBundle/Bridge/Chameleon/BackendModule/MediaManagerBackendModule.php

```diff
@@ -495,8 +499,10 @@ protected function renderList()
         $this->returnAsAjaxResponse($listReturn);
     }
 
-    private function returnGeneralErrorMessageForAjax()
+    private function returnGeneralErrorMessageForAjax(?string $logMessage = null, ?\Throwable $exception = null)
```

Comment:
The new interface in combination with the old name is somewhat missleading. `returnGeneralErrorMessageForAjax` in combination with an optional error message seems like this message would be returned via ajax - which is not the case. So we should either rename this method to indicate what the message is actually used for. Or (and I would prefer this) let the method just do what it did before and use the logger before this method is called. This makes it clear when reading code that there will be a message logged and afterwards there will be a general error returned to the user.

Reply:
changed the method name

(it is called more than 20 times...)